src/frontend/src/components/GitHubContributorCard/GitHubContributorCard.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import { Card, CardContent, Typography, Avatar, Grid } from '@material-ui/core';
+import useTelescopeContributor from '../../hooks/use-telescope-contributor';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.primary.light,
+  },
+  typography: {
+    fontSize: 13,
+  },
+  links: {
+    color: theme.palette.text.secondary,
+    textDecorationLine: 'none',
+    '&:hover': {
+      textDecorationLine: 'underline',
+    },
+    alignItems: 'flex-start',
+  },
+}));
+
+const GitHubContributorCard = () => {
+  const classes = useStyles();
+  const theme = useTheme();
+  const { contributor, error } = useTelescopeContributor();
+
+  if (contributor !== undefined && contributor !== null) {
+    return (
+      <Card className={classes.root}>
+        <CardContent>
+          <Grid container direction="row" alignItems="center">
+            <Grid item xs={2}>
+              <Avatar alt="Profile Picture" src={contributor.avatar_url} />
+            </Grid>
+            <Grid item xs={10} justify="flex-end">
+              <Typography className={classes.typography}>
+                Thank you&nbsp;
+                <a href={contributor.html_url} className={classes.links}>
+                  {contributor.login}
+                </a>
+                , for being a&nbsp;
+                <a
+                  href="https://github.com/Seneca-CDOT/telescope/graphs/contributors"
+                  className={classes.links}
+                >
+                  Telescope project contributor
+                </a>
+                , with your
+                <a
+                  href={
+                    'https://github.com/Seneca-CDOT/telescope/commits?author=' + contributor.login
+                  }
+                  className={classes.links}
+                >
+                  &nbsp;
+                  {contributor.contributions} contribution(s)
```

Comment:
Let's improve this a bit.  A lot of contributors might have only made 1 contribution, so I think calling out the number is only relevant above some threshold.  I would suggest we do:

```
// 1 or 2 contributions
contributions.

// 3 or more contributions (e.g., 8)
8 contributions.
```

The easiest way to do this might be to split out a tiny component in this same file that looks like:

```jsx
function Contributions({ contributions }) {
  return contributions < 3 ? 'contributions.' : `${contributions} contributions.`;
}

// Now use it like this:
<Contributions contributions={contributor.contributions} />
```